src/react/account/__tests__/AccountDetails.test.tsx

```diff
@@ -48,6 +48,7 @@ describe('AccountDetails', () => {
             pathname: '/accounts/bart',
           },
         },
+        oidc: { user: { id_token: 'id_token', profile: { sub: 'sub' } } },
```

Comment:
We probably could add this in the global init state for all the tests.

Reply:
Added here: https://github.com/scality/zenko-ui/pull/334/commits/bc737d8337f1d4de40258df1ab6ca2a113d7f8a3#diff-23d60af5da5b0ed4bd31d68462362b98adab06632006e787f9e74bc7eaf34790R173

Reply:
https://github.com/scality/zenko-ui/pull/334#commitcomment-72666210